src/main/msp/msp.c

```diff
@@ -1696,6 +1697,18 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         break;
 #endif
 
+#ifdef USE_SBUS_OUTPUT
+    case MSP_SBUS_OUTPUT_SETTINGS:
```

Comment:
The convention is to call it _CONFIG, not _SETTINGS

Reply:
Done